plugins/source/aws/resources/services/cloudwatch/metrics.go

```diff
@@ -0,0 +1,72 @@
+package cloudwatch
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client/tableoptions"
+	sdkTypes "github.com/cloudquery/plugin-sdk/v3/types"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+func Metrics() *schema.Table {
+	tableName := "aws_cloudwatch_metrics_custom"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_Metric.html`,
+		Resolver:    fetchCloudwatchMetrics,
+		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "logs"),
```

Comment:
I'm not sure if `logs` is the correct service here but couldn't find anything suitable in our services json.

Reply:
You want to use `monitoring` ... https://docs.aws.amazon.com/general/latest/gr/cw_region.html

Reply:
Thanks, now fixed